gen_consts.go

```diff
@@ -0,0 +1,37 @@
+// THIS FILE IS AUTOGENERATED. DO NOT EDIT.
+// Regen by running 'go generate' in the repo root.
+
+package gotgbot
+
+// The consts listed below represent all the update types that can be requested from telegram.
+const (
+	UpdateTypeMessage            = "message"
+	UpdateTypeEditedMessage      = "edited_message"
+	UpdateTypeChannelPost        = "channel_post"
+	UpdateTypeEditedChannelPost  = "edited_channel_post"
+	UpdateTypeInlineQuery        = "inline_query"
+	UpdateTypeChosenInlineResult = "chosen_inline_result"
+	UpdateTypeCallbackQuery      = "callback_query"
+	UpdateTypeShippingQuery      = "shipping_query"
+	UpdateTypePreCheckoutQuery   = "pre_checkout_query"
+	UpdateTypePoll               = "poll"
+	UpdateTypePollAnswer         = "poll_answer"
+	UpdateTypeMyChatMember       = "my_chat_member"
+	UpdateTypeChatMember         = "chat_member"
+	UpdateTypeChatJoinRequest    = "chat_join_request"
+)
+
+// The consts listed below represent all the parse_mode options that can be sent to telegram.
+const (
+	ParseModeHTML       = "HTML"
+	ParseModeMarkdownV2 = "MarkdownV2"
+	ParseModeMarkdown   = "Markdown"
+	ParseModeText       = ""
```

Comment:
How about `ParseModeNone`?

```suggestion
	ParseModeNone       = ""
```